.github/workflows/ci-additional.yml

```diff
@@ -52,6 +52,31 @@ jobs:
         shell: bash -l {0}
         run: source continuous_integration/hdfs/run_tests.sh
 
+  doctest:
+    runs-on: "ubuntu-latest"
+    env:
+      XTRATESTARGS: "--doctest-modules"
```

Comment:
It looks like right now we're running the full test suite in addition to doctests here. What do you think about adding `--ignore-glob="*test_*.py"` to skip test files so we only run doctests?

Reply:
I tried that for a while and didn't think it was working. Let me double check...

Reply:
```
pytest -v --doctest-modules --ignore-glob="*/test_*.py" dask
```

^ that works locally for me